.github/workflows/release.yaml

```diff
@@ -11,9 +11,6 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v1
-      - name: Create Helm chart
-        run: |
-          tar cvzf helm-chart.tgz helm
```

Comment:
Did we delete this because it was never being used?

Reply:
I ported this from master branch. I think @wongma7 has more context on this

Reply:
>Matthew Wong
> we have a separate github action just for releasing the helm chart, and we moved the location of the helm chart, so that one was broken. and yes, the tgz is kind of useless since we recommend using helm repo approach